### ISSUE TITLE
How to change the appearance / theme of a tab

### ISSUE THREAD

**Issue by fingolfin** [blinksh/blink#537]
Changing the theme / font / appearance via the config screen seems to only affect tabs opened afterwards, but not existing tabs. Is there any way to change the appearance of an open tab? I'd like to change between light and dark themes depending on the light conditions around me, without clothes and reopening sessions.

Thanks!

**Comment by yury** [comment]
Currently we apply themes only for new tabs because of initialization performance issues.

**Comment by Harwood** [comment]
Being able to define themes on a per host basis could be useful.

**Comment by yury** [comment]
Actually we can change color scheme right away (not fonts though).

**Comment by ryuheechul** [comment]
Yeah I just did change the font. (I'm new to Blink!) And changing font didn't make any changes for the current screen so I first though I did something wrong about loading new font (and loaded another font).
And I accidentally initiated a new screen by typing `exit`.
After that I could notice the theme I changed also got an effect just now not before.

It's understandable that it can't/won't change current tab/screen for performance reason, however I think Blink should warn users (at the time of changing font or appearance in general happens) that it will not affect current screen therefore user should initiate a new screen for that effect to be less confused.
